scripts/TestHarness/testers/RavenFramework.py

```diff
@@ -11,18 +11,21 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-from Tester import Tester
-from CSVDiffer import CSVDiffer
-from UnorderedCSVDiffer import UnorderedCSVDiffer
-from XMLDiff import XMLDiff
-from TextDiff import TextDiff
-from RAVENImageDiff import ImageDiff
-import RavenUtils
+"""
+RavenFramework is a tool to test raven inputs.
```

Comment:
RavenFramework? we said it was supposed to be removed... Just multiple differs

Reply:
Removing it will take some work, because it still does things like the modules check and handle the old testing syntax.